setup.py

```diff
@@ -121,6 +121,7 @@ def _config_exe(exe_name):
     zip_safe=False,
     setup_requires=[
         "jinja2>=2.9.3",
+        "jupyter-client==6.1.12",
```

Comment:
In principle, building NMODL doesn't require Jupyter. Once we will upload source wheels to pypi.org, building via pip will also install whole Jupyter stuff. So we should abvoid it from setup.py?

Reply:
jupyter is already a depends of nmodl for building notebooks in documentation

Reply:
discussed offline - jupyter is already installed. This merely installs different jupyter-client version due to bug.